bravado/fido_client.py

```diff
@@ -1,37 +1,59 @@
-#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
 #
-# Copyright (c) 2014, Yelp, Inc.
+# Copyright (c) 2015, Yelp, Inc.
 #
 
-"""Asynchronous HTTP client abstractions.
-"""
-
 import logging
 
 import fido
 from yelp_uri import urllib_utf8
 
-from bravado import http_client
-from bravado.multipart_response import create_multipart_content
+from bravado.mapping.http_client import APP_FORM, HttpClient
+from bravado.mapping.http_future import HttpFuture
 from bravado.mapping.param import stringify_body as param_stringify_body
+from bravado.mapping.response import ResponseLike
+from bravado.multipart_response import create_multipart_content
 
 log = logging.getLogger(__name__)
 
 
-class AsynchronousHttpClient(http_client.HttpClient):
-    """Asynchronous HTTP client implementation.
+class FidoResponseAdapter(ResponseLike):
+    """Wraps a fido.fido.Response object to provider a uniform interface
+    to the response innards.
+
+    :type requests_lib_response: :class:`fido.fido.Response`
+    """
+    def __init__(self, requests_lib_response):
+        self._delegate = requests_lib_response
+
+    @property
+    def status_code(self):
+        return self._delegate.code
+
+    @property
+    def text(self):
+        return self._delegate.body
+
+    def json(self, **_):
+        return self._delegate.json()
+
+
+class FidoClient(HttpClient):
+    """Fido HTTP client implementation.
```

Comment:
Insert "async" somewhere in the docs so it is obvious how it differs from `RequestsClient`